src/Adaptivity/Adaptivity.jl

```diff
@@ -0,0 +1,54 @@
+"""
+  Mesh Adaptivity for Gridap
+
+  Note: Despite the name, this module currently 
+        ONLY SUPPORTS HOMOGENEOUS MESH REFINEMENT!
+        The support for non-homogeneous refinement and coarsening, 
+        i.e full AMR, will be added in the future.
+"""
+module Adaptivity
+
+using FillArrays
+using LinearAlgebra
+using DataStructures
+
+using Gridap.Helpers
+using Gridap.Algebra
+using Gridap.Arrays
+using Gridap.Geometry
+using Gridap.Fields
+using Gridap.FESpaces
+using Gridap.ReferenceFEs
+using Gridap.CellData
+using Gridap.TensorValues
+using Gridap.Visualization
+
+import Base: view, *
+import Gridap.Geometry: get_grid, get_grid_topology, get_face_labeling
+import Gridap.Geometry: Triangulation, is_change_possible, best_target, get_background_model
+import Gridap.Geometry: move_contributions
+import Gridap.CellData: change_domain
+
+export RefinementRule
+export get_cell_map, get_inverse_cell_map
+
+export AdaptivityGlue
+export get_n2o_reference_coordinate_map
+
+export AdaptedDiscreteModel
+export get_model, get_parent, get_adaptivity_glue
+export is_child, is_related
+export refine
```

Comment:
We are exporting `refine`  from Adaptivity to Gridap, but NOT from Gridap to the user driver (see Exports.jl module). Should we add `refine` to the public API?

Reply:
If we do so, does it imply a change in the public API and therefore a change in the major/minor version?

Reply:
I dont think adding new methods in the public API involves changes in the minor version (assuming major is 0). 

see https://semver.org/ for some details 

In any case, once we introduce these in the public API, then we will have to change minor versions when we perform backward incompatible changes. If we feel that this dev is still experimental, we may by now defer this decision.

Reply:
I think at this early stage it is good to have the user consciously import `Gridap.Adaptivity` to be able to use the new features, don't you think?